build.gradle

```diff
@@ -244,28 +238,6 @@ subprojects {
     }
 }
 
-task composeBuild {
-    def buildTag = System.getenv('VERSION') ?: 'dev'
-    def buildPlatform = System.getenv('DOCKER_BUILD_PLATFORM') ?: 'linux/amd64'
-    def buildArch = System.getenv('DOCKER_BUILD_ARCH') ?: 'amd64'
-    def jdkVersion = System.getenv('JDK_VERSION') ?: '14.0.2'
```

Comment:
We should try not to remove the functionality we have here.

Currently, someone following the instructions on https://docs.airbyte.io/contributing-to-airbyte/developing-locally#build-with-gradle to build for M1 won't be able to since these env vars aren't captured and don't make it to the docker build steps.